Source/WebCore/platform/graphics/cg/GraphicsContextPlatformPrivateCG.h

```diff
@@ -32,7 +32,6 @@ namespace WebCore {
 
 enum class GraphicsContextCGFlag : uint8_t {
     IsLayerCGContext        = 1 << 0,
-    IsAcceleratedCGContext  = 1 << 1
 };
```

Comment:
I think this enum can be removed and `m_contextFlags` can be just a bool `m_isCALayerContext` or something else.